docs/rpc/openapi.yaml

```diff
@@ -343,39 +395,47 @@ paths:
                 $ref: ./api/trait/get-is-trait-implemented.schema.json
               example:
                 $ref: ./api/trait/get-is-trait-implemented.example.json
-    parameters:
-      - name: contract_address
-        in: path
-        required: true
-        description: Stacks address
-        schema:
-          type: string
-      - name: contract_name
-        in: path
-        required: true
-        description: Contract name
-        schema:
-          type: string
-      - name: trait_contract_address
-        in: path
-        required: true
-        description: Trait Stacks address
-        schema:
-          type: string
-      - name: trait_contract_name
-        in: path
-        required: true
-        description: Trait contract name
-        schema:
-          type: string
-      - name: trait_name
-        in: path
-        required: true
-        description: Trait name
-        schema:
-          type: string
-      - name: tip
-        in: query
-        schema:
-          type: string
-        description: The Stacks chain tip to query from
\ No newline at end of file
+      parameters:
+        - name: contract_address
+          in: path
+          required: true
+          description: Stacks address
+          schema:
+            type: string
+        - name: contract_name
+          in: path
+          required: true
+          description: Contract name
+          schema:
+            type: string
+        - name: trait_contract_address
+          in: path
+          required: true
+          description: Trait Stacks address
+          schema:
+            type: string
+        - name: trait_contract_name
+          in: path
+          required: true
+          description: Trait contract name
+          schema:
+            type: string
+        - name: trait_name
+          in: path
+          required: true
+          description: Trait name
+          schema:
+            type: string
+        - name: tip
+          in: query
+          schema:
+            type: string
+          description: The Stacks chain tip to query from
+        - name: use_latest_tip
+          in: query
+          schema:
+            type: string
```

Comment:
Thoughts on making this more OpenAPI-compliant with a `boolean` rather than a `string`? It helps with doc and client generation. 

In the API, I usually document these as `boolean` (`true`/`false`), but in the route handling code also support parsing `1`/`0`.

Reply:
sure, I'll make this change